sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -887,13 +887,12 @@ class MLPClassifier(ClassifierMixin, BaseMultilayerPerceptron):
         Whether to use early stopping to terminate training when validation
         score is not improving. If set to true, it will automatically set
         aside 10% of training data as validation and terminate training when
-        validation score is not improving by at least tol for
+        validation score is not improving by at least ``tol`` for
         ``n_iter_no_change`` consecutive epochs. The split is stratified,
         except in a multilabel setting.
         If early stopping is False, then the training stops when the training
         loss does not improve by more than tol for n_iter_no_change consecutive
         passes over the training set.
-        Only effective when solver='sgd' or 'adam'.
```

Comment:
non-related change to the PR, please revert.

Reply:
Thank you for pointing that out. I've reverted the non-related changes as per your request. Your guidance is greatly appreciated